modules/cli/src/main/java/org/apache/ignite/internal/cli/core/flow/builder/FlowBuilderImpl.java

```diff
@@ -69,6 +69,11 @@ public <OT> FlowBuilder<I, OT> then(Flow<O, OT> flow) {
         return new FlowBuilderImpl<>(this.flow.composite(flow), exceptionHandlers, decoratorRegistry);
     }
 
+    @Override
+    public <OT> FlowBuilder<I, OT> flatMap(Function<O, FlowBuilder<O, OT>> mapper) {
+        return then(it -> mapper.apply(it.value()).build().start(it));
+    }
+
     @Override
     public <OT> FlowBuilder<I, O> ifThen(Predicate<O> tester, Flow<O, OT> flow) {
```

Comment:
should we deprecate `ifThen` or something?

Reply:
Why? There are places where we use `ifThen`